src/cache_manager.cc

```diff
@@ -130,9 +122,9 @@ CacheManager::findAction(char const * action) const
 }
 
 Mgr::Action::Pointer
-CacheManager::createNamedAction(const char *actionName)
+CacheManager::createNamedAction(const SBuf &actionName)
 {
-    Must(actionName);
+    Must(!actionName.isEmpty());
```

Comment:
There is a reason.... As you noticed in the new `TypedMsgHdr.cc::getString()` this detail is coming from a possible untrusted source. This is where the semantic validation is performed on the given name. The lack of check for 0-length string was a Use-Before-Validation bug which is now fixed.

Reply:
> This is where the semantic validation is performed on the given name. The lack of check for 0-length string was a Use-Before-Validation bug which is now fixed.

If you would like to enlarge this PR scope to add new validations, you have the right to do so. The proposed Must() is not the right way to do that though. FWIW, I do not recommend enlarging this PR scope that way because it will needlessly delay its merging until we agree what should be validated, how it should be validated, and where it should be validated.